docs/team/projectPortfolio.adoc

```diff
@@ -0,0 +1,119 @@
+= Tan Yikai - Project Portfolio
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== Project: AddressBook - Level 4
+AddressBook - Level 4 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 6 kLoC.
+
+*Code contributed*: [https://github.com/CS2103AUG2017-F10-B1/main/tree/master/collated[Functional code]] [https://github.com/CS2103AUG2017-F10-B1/main/tree/master/collated[Test code]] {give links to collated code files}
```

Comment:
the code link to be the link to ur collated code file, so it shld be something like https://github.com/CS2103AUG2017-F10-B1/main/blob/master/collated/main/TanYikai.md